frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.tsx

```diff
@@ -89,6 +109,26 @@ class TagEditorParamInner extends Component<Props> {
       });
 
       setParameterValue(tag.id, null);
+
+      if (parameter.values_source_type === "static-list") {
```

Comment:
It seems that we need to / could do it for all cases without checking for `source_type`